### ISSUE TITLE
Could we have the Day-planner folder changeable?

### ISSUE THREAD

**Issue by Sigbert11** [lynchjames/obsidian-day-planner#172]
As wonderful and feature-reach this plugin already is, recently I ran into the 
- need to change the Day-planner folder 

### Situation
was surprised by there being no option to change the day-planner's folder in the settings. Did I miss out on anything?
I would imagine this being a simple thing to add on. Correct me if I am wrong.

### Reason:
All my standard data files follow some template structure with frontmatter etc. 
Day-planner files do not have this - and most probably don't need 
- ( otherwise that would then be another option to ask for  :-)   )

Thus I like to file all other files with a distinct prefix or into some special subfolders and would have to change the Day-planner's folder name.

**Comment by ldelossa** [comment]
@Sigbert11 I achieve this another **core** plugin called "Daily notes". 

Do the following:
* Enable daily notes core plugin
* Pick the folder where it will create the daily notes
* Change Day Planner to command mode
* Hit ctrl-p (or what ever your palette button is) and pick "Daily notes: open today's daily note", this will create a note with today's date (date format customizable in Daily Notes settings). 
* Hit ctrl-p again and "Day Planner: Link today's Day Planner to the current note" 

The `Day Planner` plugin now treats the note that `Daily Notes` plugin made as one it will automatically format. 

Why this is especially nice is because you can create a template of your own, tell the `Daily Notes` plugin to use it when creating the `Daily Note` for the day, and if you format it in a way that `Day Planner` understands, you can have your own custom format for your Day Planner. 

I use the following template myself: https://www.dropbox.com/s/ml913ia1ml2objz/planner.md?dl=0

**Comment by dpeter99** [comment]
Some way to not have to run the command manually would be nice.

**Comment by jun6lee** [comment]
I feel the need for this also, my top-level folder structure is very precious to be kept that way.
I'd like to put it under a subfolder if that would be viable. `/Calendars/ `

**Comment by ivan-lednev** [comment]
This is available in 0.7.0.

**Comment by hmars-t** [comment]
Where can I find this option for selecting a default folder? I'm unable to find it.

**Comment by kchasiotis** [comment]
I cannot find it either

**Comment by bwnjnOEI** [comment]
I was also puzzled for a while, but I think the author wouldn't have failed to provide this function. Then, I discovered the author's wisdom, **this plugin automatically recognizes the folder settings in the Obsidian built-in diary plugin.**

**Comment by darrendsy** [comment]
> I was also puzzled for a while, but I think the author wouldn't have failed to provide this function. Then, I discovered the author's wisdom, **this plugin automatically recognizes the folder settings in the Obsidian built-in diary plugin.**

Yes, it is, just change the location of the daily note in the core plugin setting, everything solved.
